src/bin/collapse-dtrace.rs

```diff
@@ -0,0 +1,70 @@
+use env_logger::Env;
+use std::io;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+use inferno::collapse::dtrace::{Folder, Options};
+use inferno::collapse::Collapse;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
```

Comment:
The values here don't look quite right. You probably to add a doc-comment to match [upstream](https://github.com/brendangregg/FlameGraph/blob/master/stackcollapse.pl#L3L8). If you want to be particularly helpful, you'd probably want to include some [DTrace tips](https://github.com/brendangregg/FlameGraph#dtrace) in `after_help`; if not, just delete it!

Reply:
👍